.claude/rules/governance/prompt-banned-patterns.md

```
@@ -0,0 +1,167 @@
+# Prompt Banned Patterns
+
+**"Vague prompts produce vague work."**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)